__tests__/lib/cli/envAlias.js

```diff
@@ -1,14 +1,14 @@
-/* eslint-disable jest/no-conditional-expect */
 /**
- * Internal dependencies
+ * External dependencies
  */
-import envAlias from '../../../src/lib/cli/envAlias';
+import { describe, it, expect } from '@jest/globals';
 
-const parseEnvAliasSpy = jest.spyOn( envAlias, 'parseEnvAlias' );
```

Comment:
Cannot use spyOn on a primitive value; undefined given

We used to have a weird mixture of ~Bergman and Fellini~ CJS and ESM which was ruled out by Babel. However, those hacks don't work with TypeScript.